pynonymizer/database/mssql/__init__.py

```diff
@@ -180,7 +180,7 @@ def __run_scripts(self, script_list, title=""):
             self.logger.info(results)
 
     def __create_seed_table(self, qualifier_map):
-        seed_column_lines = ["[{}] {}".format(name, _FAKE_COLUMN_TYPES[col.data_type]) for name, col in qualifier_map.items()]
```

Comment:
This will cause some weird behaviour when faker generator columns are re-used in a strategy. see how `qualifier_map` is generated and my review for a more detailed explanation.